### DISCUSSION TITLE
Possibility of querying the results using requests library Python

### DISCUSSION THREAD

**Discussion by Sangavi-123** [benbusby/whoogle-search#349] [Q&A]
I recently began exploring web-scraping. Sorry if it sounds too naive of me to ask this. Is it possible to query results for specific keywords using Python's request library and whoogle? 

I went through the readme but wasn't able to figure it out. I understand that one must create an instance first to begin using whoogle. But is there a way to get the search results?

**Comment by DUOLabs333** [comment]
Whoogle technically does something similar, but it's actually parsing and extracting the relevant bits. It may be possible to use whoogle as a library, given that it did all the hard parts already :)
